packages/react-components/react-spinner/src/components/Spinner/useSpinner.tsx

```diff
@@ -29,6 +29,7 @@ export const useSpinner_unstable = (props: SpinnerProps, ref: React.Ref<HTMLElem
         // but since it would be a breaking change to fix it, we are casting ref to it's proper type
         ref: ref as React.Ref<HTMLDivElement>,
         role,
+        tabIndex,
```

Comment:
Might make sense to not extract (is that the word? 😅) `tabIndex` at all, and just let it be part of `...rest` now that it's going on the same place